orc8r/gateway/python/magma/magmad/service_poller.py

```diff
@@ -177,7 +177,7 @@ async def _get_service_info(self):
                 )
                 self._service_info[service].continuous_timeouts = 0
             except grpc.RpcError as err:
-                logging.error(
+                logging.warning(
```

Comment:
Shall we keep the level at error but just not send it to sentry? cc @hcgatewood

Reply:
Correct. This is indeed an error (under the vanilla topology that assumes a constant Orc8r<>AGW connection), but it's not indicative that there's a bug. So let's keep at error but e.g. not send to sentry. Same goes for the other instances of inability to connect to orc8r

2 types of errors: application, platform. This is an *application* error, because something the operator's trying to do isn't working. If, on the other hand, the bootstrapper server crashed, violated an invariant, etc, then that would be a *platform* error

This whole distinction^ arises from the fact that we're providing Magma as a platform, that others then run applications on top of. So we principally care about errors that indicate a bug/breakage of the platform (for Sentry, at least)

Reply:
Thanks for explaining! I created a PR (#10349) to filter those errors via opt-in logging to Sentry. I will close this one.